week0/ujin/nginx/default.conf

```
@@ -0,0 +1,25 @@
+events { } 
+
+http {
+    upstream board_ {
```

upstream 식별자에 `_` postfix 보다는 -api postfix는 어떨까요?

```suggestion
    upstream board-api {
```